unreleased/metricstransform-fix-merge-exponential-histograms.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: "metricstransformprocessor"
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix logic in merging exponential histogram.
+
+# One or more tracking issues related to the change
+issues: [12865]
```

Comment:
Is this a bugfix? I think it's noop for the end user. So I would skip changelog here

Reply:
Oh I see the bug now. Thanks for fixing it!